src/Beberlei/Metrics/Collector/Logger.php

```diff
@@ -15,36 +15,62 @@
 
 use Psr\Log\LoggerInterface;
 
-class Logger implements Collector
+class Logger implements GaugeableCollector
 {
+    /** @var \Psr\Log\LoggerInterface */
     private $logger;
 
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     */
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function increment($variable)
     {
-        $this->logger->debug("increment:$variable");
+        $this->logger->debug('increment:' . $variable);
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function decrement($variable)
     {
-        $this->logger->debug("decrement:$variable");
+        $this->logger->debug('decrement:' . $variable);
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function timing($variable, $time)
     {
-        $this->logger->debug("timing:$variable:$time");
+        $this->logger->debug('timing:' . $variable . ':' . $time);
```

Comment:
1/ almost everywhere you are using sprintf but not here.
2/ I am not sure sprint is better than string concat, but why not
3/ here IIRC psr, spaces around the dot should be removed ?